.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You probably need this:
"extends": "angular"

for your linter angular linter to work